app/addons/activetasks/components.react.jsx

```diff
@@ -0,0 +1,387 @@
+    passesTabFilter: function (type) {
+      if (this.props.tab.toLowerCase() === type.replace('_', ' ') || this.props.tab ==='All Tasks') {
+        return true;
+      }
+      return false;
+    },
+    sortByHeader: function (e) {
+      var headerField = $(e.currentTarget),
+      columnName = headerField.attr('data-type');
+      Actions.sortByColumnHeader(columnName);
+    },
+    render: function () {
+      var collection = this.props.collection;
+      var resultsCount = 0;
+
+      return (
+        <div id="dashboard-lower-content">
+          <table className="table table-bordered table-striped active-tasks">
+            <thead>
+              <tr>
+                <th className="type"      data-type="type"        onClick={this.sortByHeader}> Type</th>
+                <th className="database"  data-type="database"    onClick={this.sortByHeader}> Database</th>
+                <th className="started"   data-type="started_on"  onClick={this.sortByHeader}> Started on</th>
+                <th className="updated"   data-type="updated_on"  onClick={this.sortByHeader}> Last updated on</th>
+                <th className="pid"       data-type="pid"         onClick={this.sortByHeader}> PID</th>
+                <th className="status"    data-type="progress"    onClick={this.sortByHeader}> Status</th>
+              </tr>
+            </thead>
+            <tbody className="js-tasks-go-here">
+              { collection.map(function (item, iteration) {
+    
+                  //'+ Filter' Tab , shows rows that match the search term, no search term shows all rows
+                  if (this.passesFilter(item)) {
+                    resultsCount++;
+                    return <ActiveTaskTableBody key={item.cid} itemInfo={this.getInfo(item)} />;
+                  }
+
+                  //Show a message if nothing is returned
+                  if ((iteration === collection.length - 1) && resultsCount === 0) {  //if we looped through everything AND there is nothing
+                    if (this.props.searchTerm ==="") {      //no search filter
+                      return (
+                        <tr className="no-matching-database-on-search" key={item.cid}>
+                          <td  colSpan="6">No active {this.props.tab} tasks.</td>
+                        </tr>
+                      );
+                    } else {  // with search filter
+                      return (  
+                        <tr className="no-matching-database-on-search" key={item.cid}>
+                          <td  colSpan="6">No active {this.props.tab} tasks match with filter: "{this.props.searchTerm}".</td>
+                        </tr>
+                      );
+                    }
+                  }
+
+                }.bind(this))
+              }
+            </tbody>
+          </table>
+        </div>
+      );
+    }
+  });
+
+  var ActiveTaskTableBody = React.createClass({
+    render: function () {
+      var data =  this.props.itemInfo;
```

Comment:
`data` is not a great variable name. Its the same as saying `var stuff` :smile:. I have no idea what this is.